### PR TITLE
Update Event class, Organization class

### PR DESCRIPTION
**Pull request by johnpryan**
- `Event` class now uses `JsonSerialziable`
- `Organization` class now uses `JsonSerializable`
- (Breaking) `Event.actor` now returns an `Actor` type instead of a
  `User`.
- Replace Repository.fromJSON with Repository.fromJson

### CODE REVIEW

**Comment by robrbecker** [comment]
Hey thanks John! Ill be able to review this tomorrow or Monday.

**Comment by johnpryan** [comment]
Any update on this?

File: lib/src/common/model/activity.dart [hunk 1]
```diff
@@ -1,42 +1,51 @@
...
 
 /// Model class for an event.
+@JsonSerializable(createToJson: false)
 class Event {
+  @JsonKey()
   Repository repo;
 
-  User actor;
+  @JsonKey()
+  Actor actor;
```
Comment (robrbecker) [inline lib/src/common/model/activity.dart]: Is it possible to keep this as a User object? This was it wouldn't be a breaking change. 
I see you added a display_login field to Actor but I can't find any reference to that being in the response anywhere.
Is that something new?

**Comment by robrbecker** [comment]
Everything else looks good, but I'm not sure about the change from a User to an Actor object. As I was reviewing I noticed there's a total inconsistency in a lot more uses of `fromJSON` or `fromJson`.

**Comment by robrbecker** [comment]
I found an Actor in the v4 github API .. but even that doesn't have a display_login https://developer.github.com/v4/interface/actor/ 🤷‍♂

**Comment by robrbecker** [comment]
FYI I've rolled (most of) these changes into the dev branch for a 6.0 release. https://github.com/SpinlockLabs/github.dart/pull/188/files#diff-916d668cf3dc57abd03a5d324ba828e7R10
I'd like to do similar changes for the rest of the model classes before releasing.

**Comment by johnpryan** [comment]
SGTM, that branch is working for me. Thanks!

**Pull request closed without merging**